scripts/release/create_release_PR_commit.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+set -eu
+
+#Versions
+#Expected to change more often , when we run this script.
+NEW_VERSION="2.15.3"
+CURRENT_VERSION="2.15.2"
+LAST_VERSION="2.15.1"
+RELEASE_BRANCH="2.15.x"
+RELEASE_ISSUE="613"
+
+# GITHUB_USERNAMES
+GH_USERNAME="pallavisontakke"
```

Comment:
can we determine this programmatically?

Reply:
Made changes to accept these GitHub parameters from CLI, when we run the script.